From f61f5f8a328207867b5c523b1c6bef6aaecaab88 Mon Sep 17 00:00:00 2001
From: Wang Mingyu <wangmy@cn.fujitsu.com>
Date: Sat, 15 Feb 2020 16:05:58 +0800
Subject: [PATCH] Bugfix: Modify the dir of install-exec-hook and
 install-data-hook

Error message:
(cd /sbin && ln -sf mount.cifs mount.smb3)
  /yocto/work001/fnst/wangmy/build/tmp/hosttools/install -c mount.cifs
'/yocto/work001/fnst/wangmy/build/tmp/work/core2-64-poky-linux/cifs-utils/6.10-r0/image/sbin'
make  install-data-hook
ln: failed to create symbolic link 'mount.smb3': Permission denied

Upstream-Status: Pending

Signed-off-by: Wang Mingyu <wangmy@cn.fujitsu.com>
---
 Makefile.am | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index fe9cd34..e9be6b3 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -119,11 +119,11 @@ endif
 SUBDIRS = contrib
 
 install-exec-hook:
-	(cd $(ROOTSBINDIR) && ln -sf mount.cifs mount.smb3)
+	(cd $(DESTDIR)$(ROOTSBINDIR) && ln -sf mount.cifs mount.smb3)
 
 install-data-hook:
-	(cd $(man8dir) && ln -sf mount.cifs.8 mount.smb3.8)
+	(test -n "$(man_MANS)" || exit 0; cd $(DESTDIR)$(man8dir) && ln -sf mount.cifs.8 mount.smb3.8)
 
 uninstall-hook:
-	(cd $(ROOTSBINDIR) && rm -f $(ROOTSBINDIR)/mount.smb3)
-	(cd $(man8dir) && rm -f $(man8dir)/mount.smb3.8)
+	(cd $(DESTDIR)$(ROOTSBINDIR) && rm -f $(DESTDIR)$(ROOTSBINDIR)/mount.smb3)
+	(test -n "$(man_MANS)" || exit 0; cd $(DESTDIR)$(man8dir) && rm -f $(DESTDIR)$(man8dir)/mount.smb3.8)
-- 
2.17.1

